packages/components/Button/package.json

```diff
@@ -28,19 +28,28 @@
   "dependencies": {
     "@uifabricshared/foundation-compose": "^1.11.8",
     "@fluentui-react-native/adapters": ">=0.8.2 <1.0.0",
+    "@fluentui-react-native/experimental-activity-indicator": "^0.5.10",
+    "@fluentui-react-native/experimental-text": ">=0.7.11 <1.0.0",
+    "@fluentui-react-native/framework": "0.7.11",
     "@fluentui-react-native/icon": "0.11.5",
     "@fluentui-react-native/interactive-hooks": ">=0.12.1 <1.0.0",
     "@fluentui-react-native/pressable": ">=0.8.17 <1.0.0",
+    "@fluentui-react-native/styling-utils": ">=0.3.4 <1.0.0",
+    "@fluentui-react-native/theme-tokens": ">=0.11.6 <1.0.0",
     "@fluentui-react-native/text": ">=0.11.8 <1.0.0",
     "@fluentui-react-native/tokens": ">=0.11.5 <1.0.0",
+    "@fluentui-react-native/use-styling": ">=0.8.2 <1.0.0",
     "@uifabricshared/foundation-composable": ">=0.10.2 <1.0.0",
-    "@uifabricshared/foundation-settings": ">=0.11.2 <1.0.0"
+    "@uifabricshared/foundation-settings": ">=0.11.2 <1.0.0",
+    "tslib": "^2.3.1"
```

Comment:
Why tslib?

Reply:
It was part of the experimental-button package. I think we ran into some sort of issue before where something we were doing (I think spread?) depended on this version of tslib